tpp-app/src/home/pages/LogSymptomsScreen.js

```diff
@@ -1,18 +1,341 @@
-import React from "react";
-import { Button, Text, View } from "react-native";
+import React, { useState, useEffect } from "react";
+import { Text, SafeAreaView, View, StyleSheet, TouchableOpacity, ScrollView, StatusBar, Alert } from "react-native";
+import CloseIcon from '../../../ios/tppapp/Images.xcassets/icons/close_icon.svg';
+import Arrow from '../../../ios/tppapp/Images.xcassets/icons/arrow.svg';
+import Accordion from "../components/Accordion";
+import { getDateString } from "../../services/utils/helpers";
+import { getCalendarByYear, getSymptomsFromCalendar } from "../../services/utils/helpers";
+import { ExerciseActivity, Symptoms } from "../../services/utils/models";
+import { GETAllTrackingPreferences } from "../../services/SettingsService";
+import { POSTsymptomsForDate } from "../../services/LogSymptomsService";
+
+
+// Alert popup constants
+const unsavedChanges = {
+  title: "Unsaved changes",
+  message: "Your changes have not been saved. Do you want to discard the changes and continue?",
+  cancelTitle: "Cancel",
+  acceptTitle: "Yes"
+}
+
+const submitError = (error) => {
+  return {
+    title: "Whoops",
+    message: error,
+    cancelTitle: "Cancel",
+    acceptTitle: "OK"
+  }
+}
+
+// Arrow component to switch dates
+const DateArrow = ({ onPress, isRight }) => {
+    const transform = { transform: [{ rotate: "180deg" }] };
+    return (
+        <TouchableOpacity onPress={onPress} style={[ styles.arrows, isRight && transform]}>
+            <Arrow />
+        </TouchableOpacity>
+    )
+}
+
+const symptoms = ['flow', 'mood', 'sleep', 'cramps', 'exercise', 'notes'];
 
 
 export default function LogSymptomsScreen({ navigation, route }) {
+  // const trackingPrefs = [true, true, true, true, true, true]; // TODO: remove
+  const trackingPrefs = GETAllTrackingPreferences();
```

Comment:
GETAllTrackingPreferences uses MultiGet() https://react-native-async-storage.github.io/async-storage/docs/api/#multiget
which will resolves looking like this I believe: 
```
resolve([[TRACK_SYMPTOMS.FLOW, true],
               [TRACK_SYMPTOMS.MOOD, true],
               [TRACK_SYMPTOMS.SLEEP, true],
               [TRACK_SYMPTOMS.CRAMPS, true],
               [TRACK_SYMPTOMS.EXERCISE, true]
]); 
```
<img width="882" alt="Screen Shot 2022-03-30 at 4 07 22 PM" src="https://user-images.githubusercontent.com/35851484/160921833-4134288c-4b8b-46cd-a44e-249d584295f2.png">

Reply:
Ah my bad, I wrote this based off the comment for `GETAllTrackingPreferences` which said it'd return an: "array of a boolean representing if a symptom is currently tracked". New changes reflect it now!